src/main/java/com/labsynch/labseer/api/ApiExportController.java

```diff
@@ -45,4 +51,20 @@ public ResponseEntity<String> molconvert(@RequestBody String json) {
 		return new ResponseEntity<String>(resultDTO.toJson(), headers, HttpStatus.OK);
 	}
 
+	@RequestMapping(value = "/lotCorpNames", method = RequestMethod.POST, headers = "Accept=application/json")
```

Comment:
The stream returns the data rather than a file location that the UI would have to go fetch from the server.  Other routes, basically write a file and then return a path to that file.